keyboards/cannonkeys/atlas/atlas.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_1x2uC( \
```

Comment:
I think eventually there will be Community Layout support for this, but the four-row version will have the same variant, so we should be specific here.

```suggestion
#define LAYOUT_ortho_5x12_1x2uC( \
```